### ISSUE TITLE
TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by ShimShamSam** [DustinJackson/html-webpack-inline-source-plugin#12]
```
 ERROR in   TypeError: Path must be a string. Received undefined

      - index.js:110 HtmlWebpackInlineSourcePlugin.processTag
        [street]/[html-webpack-inline-source-plugin]/index.js:110:32

      - index.js:45
        [street]/[html-webpack-inline-source-plugin]/index.js:45:20

      - Array.forEach

      - index.js:44 HtmlWebpackInlineSourcePlugin.processTags
        [street]/[html-webpack-inline-source-plugin]/index.js:44:19

      - index.js:25 Compilation.<anonymous>
        [street]/[html-webpack-inline-source-plugin]/index.js:25:25

      - Tapable.js:135 Compilation.applyPluginsAsyncWaterfall
        [street]/[tapable]/lib/Tapable.js:135:13

      - util.js:16 Compilation.tryCatcher
        [street]/[bluebird]/js/release/util.js:16:23


      - index.js:627
        [street]/[html-webpack-plugin]/index.js:627:12

      - index.js:150
        [street]/[html-webpack-plugin]/index.js:150:16

      - util.js:16 tryCatcher
        [street]/[bluebird]/js/release/util.js:16:23

      - promise.js:510 Promise._settlePromiseFromHandler
        [street]/[bluebird]/js/release/promise.js:510:31

      - promise.js:567 Promise._settlePromise
        [street]/[bluebird]/js/release/promise.js:567:18

      - promise.js:612 Promise._settlePromise0
        [street]/[bluebird]/js/release/promise.js:612:10

      - promise.js:691 Promise._settlePromises
        [street]/[bluebird]/js/release/promise.js:691:18

      - async.js:138 Async._drainQueue
        [street]/[bluebird]/js/release/async.js:138:16

      - async.js:148 Async._drainQueues
        [street]/[bluebird]/js/release/async.js:148:10

      - async.js:17 Immediate.Async.drainQueues
        [street]/[bluebird]/js/release/async.js:17:14
```

**Comment by ShimShamSam** [comment]
I discovered this is due to me not setting a publicPath in my webpack config. I've never had this issue before. Should you maybe default to '/' if one isn't defined?

**Comment by weerd** [comment]
I ran into this issue as well. Thanks @ShimShamSam for indicating what was causing it. Setting a `output.publicPath` helped resolve it, but I agree that maybe the plugin should have a default, or defer to the `output.path` if no public path is define?

**Comment by d-bitz** [comment]
Should be fixed by 22a7af61235b4c8b20005aacbd791f5da918cd7c.

**Comment by weerd** [comment]
Nice, thanks @DustinJackson :)
